src/PathFinding/Hexagon.cpp

```diff
@@ -146,22 +147,40 @@ void Hexagon::setInRender(bool value)
 
 unsigned int Hexagon::orientationTo(Hexagon* hexagon)
 {
-    int dx = hexagon->cubeX() - cubeX();
-    int dy = hexagon->cubeY() - cubeY();
 
-    if (dx == 0) // 0 || 3
+    int x1 = x();
+    int y1 = y();
+    int x2 = hexagon->x();
+    int y2 = hexagon->y();
+    int dx = x2 - x1;
+    int dy = y2 - y1;
+
+    unsigned int result = 0;
+
+    if (dx)
     {
-        return dy > 0 ? 0 : 3;
+        // triginometry magick.
+        // basically, we try to find angle to second hex in circle, where first hex is center
+        // and then find out to which of 60� slices it belongs
+
+        double degree = atan2((double)-dy, (double)dx) * 180.0 * 0.3183098862851122; //  180 * 1/PI
```

Comment:
There's non-standart M_1_PI defined in math.h, but it differs in value from original exe.
It's the only place it's used, so there's no need move it somewhere as const double.
I even left the comment describing it, duh.